packages/react/src/primitives/Select/Select.tsx

```diff
@@ -65,15 +69,17 @@ const SelectPrimitive: Primitive<SelectProps, 'select'> = (
         {placeholder && <option value="">{placeholder}</option>}
         {children}
       </View>
-      <Flex
-        className={classNames(
-          ComponentClassNames.SelectIconWrapper,
-          classNameModifier(ComponentClassNames.SelectIconWrapper, size)
-        )}
-        color={iconColor}
-      >
-        {icon ?? icons?.expand ?? <IconExpandMore />}
-      </Flex>
+      {isMultiple || selectSize! > 1 ? null : (
```

Comment:
Could we provide a default value for selectSize instead of relaxing the not null check? What happens here if selectSize is not defined?

Reply:
+1. Using a default of `0` would make the most sense IMO

Reply:
Here is the [docs] I was referencing to (https://www.w3schools.com/tags/att_size.asp), in the demo case our current placeholder is a part of the dropdown, that is shown when a `selectSize` is 1

Reply:
So if i'm understanding correctly, to prevent a breaking change the default `selectSize` should be `1`, and we should show the placeholder when the `selectSize` is `1` or greater right?

Reply:
Yeap, that's my understanding of it as well